include/rospack/rospack.h

```diff
@@ -285,6 +285,12 @@ class ROSPACK_DECL Rosstackage
                   std::string& path);
 
     /**
+     * @brief List license(s) of a stackage whose name is passed.
+     * @param deps Names of the depended stackages.
+     * @param licenses Pairs of (package name, license(s)).
+     */
+    void licenses(std::vector<std::string>& deps, std::set<std::pair<std::string, std::vector<std::string> > >& licenses);
```

Comment:
So, this one doesn't change ABI I think. I ran abi compatibility check against the current DEB and the one compiled using this PR.

![abicc_librospack_indigo_2 2 7_with_2 2 8-1](https://cloud.githubusercontent.com/assets/1840401/23487772/1db96094-fe9d-11e6-96d7-cbe66ee9948d.png)

```
$ dpkg -p ros-indigo-rospack|grep Ver
Version: 2.2.7-0trusty-20160627-160632-0700
```
```
$ git log     
commit 5af0cc2a3d36f14a9de450ac28dbe9c89704dad8
Author: Isaac I.Y. Saito <130s@2000.jukuin.keio.ac.jp>
Date:   Sat Feb 6 15:33:44 2016 -0800

    Add license feature
$  catkin b rospack --no-deps
```